src/content/en/2020/http2.md

```diff
@@ -180,25 +241,28 @@ Unfortunately, the upgrade path for servers is more difficult, especially with t
+  description="A bar chart comparing the percentge of websites served over HTTP/2 between 2019 and 2020. Cloudflare increased to 93.08% from 85.40%. Litespeed increased to 81.91% from 70.80%. Openresty increased to 66.24% from 51.40%. Nginx increased to 60.84% from 49.20%. Apache increased to 27.19% from 18.10% and MIcorsoft-IIS increased to 22.82% from 14.10%.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSOkWXtrbMfTLdhlKbBGDjRU3zKbnCQi3iPhfuKaFs5mj4smEzInDCYEnk63gBdgsJ3GFk2gf4FOKCU/pubchart?oid=936321266&format=interactive",
   sheets_gid="306338094",
   sql_file="count_of_h2_and_h3_sites_grouped_by_server.sql"
   )
 }}
 
-A long-term recommendation to improve website performance has been to use a Content Delivery Network (CDN). The benefits are a reduction in latency by both serving content and terminating connections closer to the end user. This helps mitigate the rapid evolution in protocol deployment and the additional complexities in tuning servers and operating systems (see the prioritization section for more details for HTTP/2). To utilise the new protocols effectively, using a CDN can be seen as the recommended approach.
+A long-term recommendation to improve website performance has been to use a CDN. The benefit is a reduction in latency by both serving content and terminating connections closer to the end user. This helps mitigate the rapid evolution in protocol deployment and the additional complexities in tuning servers and operating systems (see the [Prioritization](#prioritization) section for more details). To utilize the new protocols effectively, using a CDN can be seen as the recommended approach.
+
+CDNs can be classed in two broad categories: those that serve the home page and/or asset subdomains, and those that are mainly used to serve third-party content. Examples of the first category are the larger generic CDNs (such as Cloudflare, Akamai, or Fastly) and the more specific (such as WordPress or Netlify). Looking at the difference in HTTP/2 adoption rates for home pages served with or without a CDN, we see:
 
-CDNs can be classed in two broad categories. The first category directly serves the home page and/or asset subdomains. Examples are the larger generic CDNs (such as Cloudflare, Akamai, or Fastly) and the more specific (such as Wordpress or Netlify). Looking at the difference in HTTP/2 adoption rates for home pages served with or without a CDN, we see:
 
 - **80%** of mobile home pages are served over HTTP/2 if a CDN is used
 - **30%** of mobile home pages are served over HTTP/2 if a CDN is not used
 
+{# TODO(authors, editors): Throughout the chapter there are several lead-ins to figures that are very similar/redundant with the existing figure captions. Is it worth writing these out? #}
```

Comment:
@dotjs

Reply:
Agreed they were written before adding the captions. I will review.